README.rst

```diff
@@ -13,18 +13,11 @@ Requirements
 Installation guide
 ------------------
 
+You can install it by using CLI just have next command executed
```

Comment:
I've just realized that there should be one `\n` before / after rst `..code-block::`'s. Currently our markup has been broken a bit... Can you please fix it?